README.md

```diff
@@ -16,7 +16,10 @@ This is a [WordPress core development](https://core.trac.wordpress.org/browser/t
 * Install Lando via a [GitHub DMG file](https://github.com/lando/lando/releases) or via [Homebrew Cask](http://caskroom.io/) (`brew cask install lando`). You also need to have Docker installed (if you install Lando with Homebrew Cask, it will automatically take care of that for you).
 * Clone this repository into a directory of your choice. Navigate to that directory.
 * Run `lando start`. When doing this for the first time, it will set the environment up for you, so it will take a bit longer than on subsequent starts.
-* Run `lando wp-install`. This will download the WordPress core development repository, install necessary dependencies, create a first build, and install WordPress for you.
-* Access your site under `https://wordpressdev.lndo.site/build/`.
+* Access your site under `https://wordpressdev.lndo.site/`. If you're having trouble connecting, you may be facing the [DNS Rebinding Protection issue](https://docs.devwithlando.io/issues/dns-rebind.html). To fix, and to ensure you can develop while offline, follow the [Working Offline](https://docs.devwithlando.io/config/proxy.html#working-offline-or-using-custom-domains) steps. In other words, add the folliwng to your host machine's `/etc/hosts` file:
+```
+127.0.0.1       wordpressdev.lndo.site
+```
+* ~Work with themes and plugins under `core/build/wp-content`. The `core/src` is only for WordPress core sources. You work in the build version of core since core can no longer be run from src.~ (Do not do this because content will be deleted when doing a new build!)
```

Comment:
Is this relevant anymore? Should this not advise to develop core themes in `app/public/trunk-git/src` but to develop non-core themes in `public/content`?

Reply:
Yep, I had taken a note to update the docs accordingly. Will do that.